nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -25,59 +26,116 @@ class MLTuning(ExperimentFunction):
+                                     regressor="decision_tree",
+                                     alpha=1.0, learning_rate="no", 
+                                     activation="no", solver="no"), parametrization)
+            self.evaluation_function = partial(self._ml_parametrization,  # type: ignore
+                                               noise_free=True, criterion="mse", 
+                                               dimension=dimension, min_samples_split=0.00001,
+                                               regressor="decision_tree",        
+                                               alpha=1.0, learning_rate="no", 
+                                               activation="no", solver="no")
+        elif regressor == "any":
+            parametrization = p.Instrumentation(
+                depth=p.Scalar(lower=1, upper=1200).set_integer_casting(),
+                criterion=p.Choice(["mse", "friedman_mse", "mae"]),
+                min_samples_split=p.Log(lower=0.0000001, upper=1),
+                regressor=p.Choice(["mlp", "decision_tree"]),
+                activation=p.Choice(["identity", "logistic", "tanh", "relu"]),
+                solver=p.Choice(["lbfgs", "sgd", "adam"]),
+                learning_rate=p.Choice(["constant", "invscaling", "adaptive"]),
+                alpha=p.Log(lower=0.0000001, upper=1.),
+            )        
+            super().__init__(partial(self._ml_parametrization, dimension=dimension, noise_free=False), parametrization)
+            self.evaluation_function = partial(self._ml_parametrization, dimension=dimension, noise_free=True)  # type: ignore
+        elif regressor == "decision_tree_regression":
             # Adding criterion{“mse”, “friedman_mse”, “mae”}
             parametrization = p.Instrumentation(
                 depth=p.Scalar(lower=1, upper=1200).set_integer_casting(),
                 criterion=p.Choice(["mse", "friedman_mse", "mae"]),
-                min_samples_split=p.Scalar(lower=0.0000001, upper=1)
+                min_samples_split=p.Log(lower=0.0000001, upper=1),
+                regressor="decision_tree",
+            )        
+            super().__init__(partial(self._ml_parametrization, dimension=dimension, noise_free=False,        
+                                     alpha=1.0, learning_rate="no", regressor="decision_tree", 
+                                     activation="no", solver="no"), parametrization)
+            self.evaluation_function = partial(self._ml_parametrization, dimension=dimension, criterion="mse",  # type: ignore
+                                               min_samples_split=0.00001,
+                                               regressor="decision_tree", noise_free=True,        
+                                               alpha=1.0, learning_rate="no", 
+                                               activation="no", solver="no")
+        elif regressor == "nn":
```

Comment:
why not `mlp` if that's what you provide to `regressor`? that would make it more uniform

Reply:
Right. I switch to mlp everywhere.